### PR TITLE
879456 - Update the code snippet for the TabHeaderPadding property in SfTabView. [development]

### PR DESCRIPTION
**Pull request by Arunachalam3641**
Update the code snippet for the TabHeaderPadding property in SfTabView.

### CODE REVIEW

File: MAUI/TabView/Tab-Item-Customization.md [hunk 1]
```diff
@@ -441,14 +441,12 @@ N> The `TabHeaderPadding` property is only applicable when [TabWidthMode](https:
 {% tabs %}
 
 {% highlight xaml %}
-<tabView:SfTabView>
-    <tabView:SfTabItem TabWidthMode="SizeToContent" TabHeaderPadding="5,10,5,10">
+<tabView:SfTabView TabWidthMode="SizeToContent" TabHeaderPadding="5,10,5,10">
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Tab-Item-Customization.md:444]: Why did you remove the tab item? just move the properties to the tab view.

Comment (Arunachalam3641) [reply MAUI/TabView/Tab-Item-Customization.md:444]: Now, I added the tab item.

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]
